packaging/hip-targets-release.cmake

```diff
@@ -30,15 +30,14 @@ list(APPEND _IMPORT_CHECK_FILES_FOR_hip::hip_hcc_static "${_IMPORT_PREFIX}/lib/l
 
 # Import target "hip::hip_hcc" for configuration "Release"
 set_property(TARGET hip::hip_hcc APPEND PROPERTY IMPORTED_CONFIGURATIONS RELEASE)
-if(HIP_COMPILER STREQUAL "hcc" OR HIP_COMPILER STREQUAL "clang")
+if(CMAKE_CXX_COMPILER MATCHES "(.*)hcc" OR CMAKE_CXX_COMPILER MATCHES "(.*)clang(.*)")
 set_target_properties(hip::hip_hcc PROPERTIES
   IMPORTED_LINK_INTERFACE_LIBRARIES_RELEASE "hcc::hccrt;hcc::hc_am"
   IMPORTED_LOCATION_RELEASE "${_IMPORT_PREFIX}/lib/libhip_hcc.so"
   IMPORTED_SONAME_RELEASE "libhip_hcc.so"
   )
 else()
 set_target_properties(hip::hip_hcc PROPERTIES
-  IMPORTED_LINK_INTERFACE_LIBRARIES_RELEASE "hcc::hccrt"
   IMPORTED_LOCATION_RELEASE "${_IMPORT_PREFIX}/lib/libhip_hcc.so"
   IMPORTED_SONAME_RELEASE "libhip_hcc.so"
   )
```

Comment:
The following `hip::hip_hcc` properties are unconditional and might be brought out of `if() ... else()`:
```cmake  
IMPORTED_LOCATION_RELEASE "${_IMPORT_PREFIX}/lib/libhip_hcc.so"
IMPORTED_SONAME_RELEASE "libhip_hcc.so"
```

Reply:
Actually, `IMPORTED_LINK_INTERFACE_LIBRARIES_RELEASE "hcc::hccrt;hcc::hc_am"` shouldnt be there either, so the whole `if` statement can be removed.